File: app/src/main/java/com/wealoha/social/beans/PostCommentNotify2.java
package com.wealoha.social.beans;

import com.wealoha.social.api.user.bean.User2;

/**
 * 
 * 
 * @author javamonk
 * @createTime 2015年2月25日 下午12:25:04
 */
public class PostCommentNotify2 extends AbsNotify2 {

	private static final long serialVersionUID = 6159518661888749040L;
	private final boolean replyMe;
	private final String comment;
	private final String commentId;
	private final User2 fromUser2;
	private final Post post;
	private int count;

	public PostCommentNotify2(Notify2Type type, boolean unread, String notifyid, long updateTimeMillis, boolean replyMe, String comment, String commentid, User2 fromUser2, Post post, int count) {
		super(type, unread, notifyid, updateTimeMillis);
		this.replyMe = replyMe;
		this.comment = comment;
		this.commentId = commentid;
		this.fromUser2 = fromUser2;
		this.post = post;
		this.count = count;
	}

	public int getCount() {
		return count;
	}

	public void setCount(int count) {
		this.count = count;
	}

	public boolean isReplyMe() {
		return replyMe;
	}

	public String getCommentId() {
		return commentId;
	}

	public String getComment() {
		return comment;
	}

	public User2 getFromUser2() {
		return fromUser2;
	}

	public Post getPost() {
		return post;
	}

}

File: app/src/main/java/com/wealoha/social/beans/PostCommentNotify2.kt
package com.wealoha.social.beans

/**
 *
 *
 * @author javamonk
 * @createTime 2015年2月25日 下午12:25:04
 */
class PostCommentNotify2(
    type: Notify2Type,
    unread: Boolean,
    notifyid: String?,
    updateTimeMillis: Long,
    private val replyMe: Boolean,
    private val comment: String,
    private val commentId: String,
    private val fromUser2: User2,
    private val post: Post,
    private var count: Int
) : AbsNotify2(type, unread, notifyid, updateTimeMillis) {
    fun getCount(): Int {
        return count
    }

    fun setCount(count: Int) {
        this.count = count
    }

    fun isReplyMe(): Boolean {
        return replyMe
    }

    fun getCommentId(): String {
        return commentId
    }

    fun getComment(): String {
        return comment
    }

    fun getFromUser2(): User2 {
        return fromUser2
    }

    fun getPost(): Post {
        return post
    }

    companion object {
        private const val serialVersionUID = 6159518661888749040L
    }

}